tests/integration/users.test.js

```diff
@@ -696,26 +686,15 @@ describe('GET /users/learners', () => {
         expect(response.statusCode).toBe(role.expectedCode);
       });
     });
-
-    it('should return 403 as user is client_admin and does not request user from his company', async () => {
-      authToken = await getToken('client_admin');
-      const response = await app.inject({
-        method: 'GET',
-        url: `/users/learners?company=${otherCompany._id}`,
-        headers: { Cookie: `alenvi_token=${authToken}` },
-      });
-
-      expect(response.statusCode).toBe(403);
-    });
   });
 });
 
 describe('GET /users/active', () => {
   let authToken;
-  describe('CLIENT_ADMIN', () => {
+  describe('COACH', () => {
```

Comment:
est ce que ce test `it('should get all active auxiliary users (company B)', async () => {` a du sens ? pour moi il est redondant avec celui sur _company A_

Reply:
done